src/token/feature-ext-parser.js

```diff
@@ -0,0 +1,27 @@
+
+module.exports = function featureExtAckParser(parser, colMetadata, options, callback) {
```

Comment:
I don't think this parser is quite right. 🤔 

According to the MS-TDS spec, the `FEATUREEXTACK` token can return multiple different features, while this implementation only ever returns the latest feature that was read.

Also, it only returns the feature data length, not the actual data itself.

What do you think about changing the token to return an object with the following structure:

```
{
  name: 'FEATUREEXTACK',
  event: 'featureExtAck',
  features: [
    {
      id: number,
      data: Buffer,
    },
    ....
  ]
}
```